R/result.R

```diff
@@ -23,10 +23,16 @@ result <- function(x, message = NULL, correct = FALSE) {
 }
 
 
-# TODO-document
-# Should be what EVERY check_* returns
+#' Graded submission value
+#' \code{grader_graded} object
+#'
+#' returned by every \code{check_*} function.
```

Comment:
"The return value from \code{graded} should be returned by every \code{*-check} chunk when used with \code{link{grade_learnr}}."